src/Tizen.Sensor/Tizen.Sensor/EventArgs/HeartRateMonitorBatchDataUpdatedEventArgs.cs

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
```

Comment:
I am not sure about the given year of the license header. If it is a new API it should be 2020.